ui-tests/src/test/java/org/openmrs/reference/page/ManageFormsPage.java

```diff
@@ -53,13 +53,14 @@ public void addIcon(String icon) {
         driver.findElement(EXTENSION_FORM_ICON).sendKeys(icon);
     }
 
-    public boolean addPresent() {
```

Comment:
you probably don't intend to have these changes in here, do you?

Reply:
Sorry will revert this